extensions/wikia/Insights/templates/Insights_flagsFiltering.php

```diff
@@ -1,15 +1,13 @@
 <div class="insights-header-sorting">
-	<form class="insights-sorting-form" method="GET">
-		<label for="sort"><?= wfMessage( 'insights-flags-filter-label' )->escaped(); ?></label>
-		<select class="insights-sorting" name="flagTypeId" onchange="this.form.submit()">
-			<?php
-			foreach ( $flagTypes as $flagType ):
-				$selected = ( $flagType['flag_type_id'] == $selectedFlagTypeId ) ? true : false;
-			?>
-				<option value="<?= $flagType['flag_type_id']; ?>" <?= $selected ? 'selected="selected"' : ''; ?>><?= $flagType['flag_name'] ?></option>
-			<?php
-			endforeach;
-			?>
-		</select>
-	</form>
+	<label for="sort"><?= wfMessage( 'insights-flags-filter-label' )->escaped(); ?></label>
+	<select class="insights-sorting" name="flagTypeId" onchange="this.form.submit()">
+		<?php
+		foreach ( $flagTypes as $flagType ):
+			$selected = ( $flagType['flag_type_id'] == $selectedFlagTypeId ) ? true : false;
```

Comment:
You do not need ternary operator here - `$selected = $flagType['flag_type_id'] === $selectedFlagTypeId` will be enough. :)